### PR TITLE
edit booking form #83

### PR DESCRIPTION
**Pull request by Hanan795**
- read data from props
- edit validation to all field
- validation message appearance
- back button should return to available rooms component

### CODE REVIEW

File: client/src/components/Common/BookingForm/BookingForm.js [hunk 1]
```diff
@@ -4,34 +4,45 @@ import Clock from "../../../Images/clock_icon.png";
...
+
 class BookingForm extends Component {
   constructor(props) {
-    super(props);
+    super(props); 
     this.state = {
       userId: 3,
-      date: "2020-05-23T11:00:00.000Z",
+      date: '',
       roomId: 5,
```
Comment (bayanseder) [inline client/src/components/Common/BookingForm/BookingForm.js]: should read the room Id from props with the room name

**Review by bayanseder — APPROVED** [review]

File: client/src/components/Common/BookingForm/BookingForm.js (hunk 1 above)
Comment (Hanan795) [reply client/src/components/Common/BookingForm/BookingForm.js]: I forget that. Hmm so we should put the navbar in userhome component so user id can be passed to rooms component then to bookingForm component @yaqootturman

File: client/src/components/Common/EditEvent/EditEvent.js [hunk 2]
```diff
@@ -1,38 +1,53 @@
...
       eventDescription: event.event_description,
       eventNote: event.event_note,
+      date: date,
+      roomId: event.room_id,
     });
+    axios
+      .get(`/api/rooms`)
+      .then((res) => this.setState({ rooms: res.data })).then(() => {
+        this.state.rooms = this.state.rooms.filter(room => room.room_id === this.state.roomId)
+        this.setState({ room_name: this.state.rooms[0].room_name })
```
Comment (yaqootturman) [inline client/src/components/Common/EditEvent/EditEvent.js]: The room name still an empty

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: client/src/components/Common/EditEvent/EditEvent.js (hunk 2 above)
Comment (Hanan795) [reply client/src/components/Common/EditEvent/EditEvent.js]: no, it display it, but take little time for that

**Review by tasneembhiri — APPROVED** [review]

**Review by tasneembhiri — APPROVED** [review]

**Review by yaqootturman — APPROVED** [review]

**Pull request merged**